docs/go-configuration.md

```diff
@@ -4,27 +4,599 @@ title: Scope Go Agent configuration
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Environment Variable-->
+
+```sh
+SCOPE_INSTRUMENTATION_DB_STATEMENT_VALUES=true
+```
+
+<!--Testing instrumentation-->
+
+```go
+import (
+    "os"
+    "testing"
+
+    "github.com/lib/pq"
+    "github.com/go-sql-driver/mysql"
+
+    "go.undefinedlabs.com/scopeagent"
+    scopesql "go.undefinedlabs.com/scopeagent/instrumentation/sql"
+)
+
+func TestMain(m *testing.M) {
+    sql.Register("scope-postgres", scopesql.WrapDriver(&pq.Driver{}, scopesql.WithStatementValues()))
+    sql.Register("scope-mysql", scopesql.WrapDriver(&mysql.MySQLDriver{}, scopesql.WithStatementValues()))
+
+    os.Exit(scopeagent.Run(m))
+}
+
+func TestExample(t *testing.T) {
+    // ...
+}
+
+func BenchmarkExample(b *testing.B) {
+    // ...
+}
+```
+
+<!--Runtime instrumentation-->
+
+```go
+import (
+    "database/sql"
+    
+    "github.com/lib/pq"
+    "github.com/go-sql-driver/mysql"
+
+    scopesql "go.undefinedlabs.com/scopeagent/instrumentation/sql"
+)
+
+func main() {
+    // ...
+
+    sql.Register("scope-postgres", scopesql.WrapDriver(&pq.Driver{}, scopesql.WithStatementValues()))
+    sql.Register("scope-mysql", scopesql.WrapDriver(&mysql.MySQLDriver{}, scopesql.WithStatementValues()))
+
+    // ...
+}
+
+// ...
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Include HTTP payloads in HTTP span tags
+
+You can include HTTP payloads in the related HTTP spans, truncated to the first 512 bytes.
+
+By default, Scope Go Agent will not send HTTP payloads as they are considered sensitive information.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Environment Variable-->
+
+```sh
+SCOPE_INSTRUMENTATION_HTTP_PAYLOADS=true
+```
+
+<!--Testing instrumentation-->
+
+```go
+import (
+    "os"
+    "testing"
+
+    "go.undefinedlabs.com/scopeagent"
+    "go.undefinedlabs.com/scopeagent/instrumentation/nethttp"
+)
+
+func TestMain(m *testing.M) {
+    // Default Http Client
+	nethttp.PatchHttpDefaultClient(nethttp.WithPayloadInstrumentation())
+
+    os.Exit(scopeagent.Run(m))
+}
+
+func TestExample(t *testing.T) {
+    // Custom http.Client instance
+    client := &http.Client{Transport: &nethttp.Transport{ PayloadInstrumentation: true }}
+
+    // ...
+}
+
+func BenchmarkExample(b *testing.B) {
+    // ...
+}
+```
+
+<!--Runtime instrumentation-->
+
+```go
+import (
+    "go.undefinedlabs.com/scopeagent/instrumentation/nethttp"
+)
+
+func main() {
+    // Default Http Client
+    nethttp.PatchHttpDefaultClient(nethttp.WithPayloadInstrumentation())
+
+    // Custom http.Client instance
+    client := &http.Client{Transport: &nethttp.Transport{ PayloadInstrumentation: true }}
+
+    // ...
+}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Changing the log folder
+
+By default, the log folder were the Scope agent stores all the execution logs depends on the platform running:
+
+```
+Linux = /var/log/scope
+Windows = ${HOME}/AppData/Roaming/scope/logs
+MacOs = ${HOME}/Library/Logs/Scope
+```
```

Comment:
Not sure why this is a code block. Just do a markdown list instead:

```md
* Linux: `/var/log/scope`
* Windows: `${HOME}/AppData/Roaming/scope/logs`
* Mac OS: `${HOME}/Library/Logs/Scope`

Reply:
ok